app/models/slide.rb

```diff
@@ -154,20 +156,28 @@ def validate_background_file
     end
 
     def validate_video_file(type, opts = nil)
-      video = File.open(self.send("#{type}").file.path)
-
-      if video.size > 12.megabytes
-        errors.add("#{type} Video: ", 'You cannot upload a video larger than 12 MB')
+      if !self.send("#{type}").file.nil?
+        video = File.open(self.send("#{type}").file.path)
```

Comment:
This `self.send("#{type}")` construct seems odd to me. I wonder if there's a better way to do this. But since it's already in use and seems to work, I'm ok leaving it be.